virtual_rainforest/models/animals/animal_cohorts.py

```diff
@@ -303,3 +307,42 @@ def can_reproduce(self) -> bool:
 
         """
         return self.stored_energy >= self.reproduction_energy_threshold
+
+    def is_below_energy_threshold(self) -> bool:
+        """Check if cohort's energy is below a certain threshold.
+
+        Return:
+            A bool of whether the current energy state is above the migration threshold.
+        """
+        return (
+            self.stored_energy
+            < ENERGY_PERCENTILE_THRESHOLD * self.reproduction_energy_threshold
```

Comment:
Ahh the use of `self.reproduction_energy_threshold` made me think that this was for migratory birds, is reproduction energy just used as a convenient stand in?

Reply:
It is just a stand-in. I'll update the docs to be explicit.